core/server/data/timezones.json

```diff
@@ -2,328 +2,263 @@
 	"timezones": [
 		{
 			"name": "Pacific/Pago_Pago",
-			"label": "(GMT -11:00) Midway Island, Samoa",
-			"offset": -660
+			"label": "(GMT -11:00) Midway Island, Samoa"
```

Comment:
We're keeping it for now, later on it will likely be removed and the client will auto-calculate the offset for display.